tools/cli/pkg/command/upgrade_cluster.go

```diff
@@ -0,0 +1,60 @@
+package command
+
+import (
+	"fmt"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/common/orchestration"
+	"github.com/kyma-project/control-plane/tools/cli/pkg/logger"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type UpgradeClusterCommand struct {
+	UpgradeCommand
+	cobraCmd *cobra.Command
+}
+
+func NewUpgradeClusterCommand() *cobra.Command {
+	cmd := UpgradeClusterCommand{UpgradeCommand: UpgradeCommand{}}
+	cobraCmd := &cobra.Command{
+		Use:   "cluster --target {TARGET SPEC} ... [--target-exclude {TARGET SPEC} ...]",
+		Short: "Upgrades Kubernetes cluster on one or more Kyma Runtimes.",
+		Long: `Upgrade Kubernetes cluster on targets of Runtimes.
+The upgrade is performed by Kyma Control Plane (KCP) within a new orchestration asynchronously. The ID of the orchestration is returned by the command upon success.
+The targets of Runtimes are specified via the --target and --target-exclude options. At least one --target must be specified.
+The version of Kubernetes is configured by Kyma Environment Broker (KEB).
+Additional Kyma configurations to use for the upgrade are taken from Kyma Control Plane during the processing of the orchestration.`,
+		Example: `kcp upgrade cluster --target all --schedule maintenancewindow    Upgrade Kubernetes cluster on Runtime in their next respective maintenance window hours.
```

Comment:
Replace all tabs with spaces in example, to be consistent with other commands. Indent the explanations into the same position

Reply:
My bad, did it as @fhivemind also mentioned